### PR TITLE
890013 - WPF GanttControl review correction changes

### PR DESCRIPTION
**Pull request by Kerubananthan3761**
## Description
Addressed GanttControl review correction changes

### CODE REVIEW

File: wpf/Gantt/migrate-from-sftreegrid.md [hunk 1]
```diff
@@ -117,33 +123,23 @@ To facilitate the migration from [`GridTreeControl`](https://help.syncfusion.com
...
-<td>{{'[EventHandler(NodeCollapsedEventArgs)](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.NodeCollapsedEventArgs.html)'| markdownify }}</td>
-<td>{{'[NodeCollapsed](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.SfTreeGrid.html#Syncfusion_UI_Xaml_TreeGrid_SfTreeGrid_NodeCollapsed)'| markdownify }}</td>
+<td>{{'[EventHandler(NodeExpandedEventArgs)](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.NodeExpandedEventArgs.html)'| markdownify }}</br>
+{{'[EventHandler(NodeCollapsedEventArgs)](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.NodeCollapsedEventArgs.html)'| markdownify }}</td>
+<td>{{'[NodeExpanded](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.SfTreeGrid.html#Syncfusion_UI_Xaml_TreeGrid_SfTreeGrid_NodeExpanded)'| markdownify }}</br>
+{{'[NodeCollapsed](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.SfTreeGrid.html#Syncfusion_UI_Xaml_TreeGrid_SfTreeGrid_NodeCollapsed)'| markdownify }}</td>
 </tr>
 
 <tr>
 <td>{{'[GridTreeNodeEventHandler](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Grid.GridTreeNodeEventHandler.html)'| markdownify }}</td>
```
Comment (Karthickmani97) [inline wpf/Gantt/migrate-from-sftreegrid.md:133]: Change handler to `GridTreeNodeCancelEventHandler`

File: wpf/Gantt/migrate-from-sftreegrid.md [hunk 2]
```diff
@@ -117,33 +123,23 @@ To facilitate the migration from [`GridTreeControl`](https://help.syncfusion.com
...
 <td>{{'[ExpandStateChanging](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Grid.GridTreeControl.html#Syncfusion_Windows_Controls_Grid_GridTreeControl_ExpandStateChanging)'| markdownify }}</td>
-<td>{{'[EventHandler(NodeExpandedEventArgs)](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.NodeExpandedEventArgs.html)'| markdownify }}</td>
-<td>{{'[NodeExpanding](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.SfTreeGrid.html#Syncfusion_UI_Xaml_TreeGrid_SfTreeGrid_NodeExpanding)'| markdownify }}</td>
+<td>{{'[EventHandler(NodeExpandedEventArgs)](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.NodeExpandedEventArgs.html)'| markdownify }}</br>
+{{'[EventHandler(NodeCollapsedEventArgs)](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.NodeCollapsedEventArgs.html)'| markdownify }}</td>
```
Comment (Karthickmani97) [inline wpf/Gantt/migrate-from-sftreegrid.md:136]: Update `NodeExpandingEventArgs`

`NodeCollapsingEventArgs`

File: wpf/Gantt/migrate-from-sftreegrid.md [hunk 3]
```diff
@@ -359,4 +387,201 @@ To facilitate the migration from [`GridTreeControl`](https://help.syncfusion.com
...
+Syncfusion.GridCommon.WPF
+</td>
+<td>Syncfusion.SfGrid.WPF</td>
+</tr>
+</table>
+
+### Alternate events
+<table>
+<tr>
+<th>GridTree control Structure</th>
```
Comment (Karthickmani97) [inline wpf/Gantt/migrate-from-sftreegrid.md:413]: Rename to `GridTreeControl` in all places

File: wpf/Gantt/migrate-from-sftreegrid.md (hunk 1 above)
Comment (Kerubananthan3761) [reply wpf/Gantt/migrate-from-sftreegrid.md]: GridTreeNodeCancelEventHandler - changed.

File: wpf/Gantt/migrate-from-sftreegrid.md (hunk 2 above)
Comment (Kerubananthan3761) [reply wpf/Gantt/migrate-from-sftreegrid.md]: Updated NodeExpandingEventArgs, NodeCollapsingEventArgs

File: wpf/Gantt/migrate-from-sftreegrid.md [hunk 4]
```diff
@@ -162,16 +158,48 @@ To facilitate the migration from [`GridTreeControl`](https://help.syncfusion.com
...
+Metro</br>
+Blend</br>
+VS2010
+</td>
+<td>Nil</td>
+<td>Nil</td>
+</tr>
+
+<tr>
+<td>{{'[GridSelectionMode](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Gantt.GridSelectionMode.html)'| markdownify }}</td>
```
[the remark this answers is not in the document]
Comment (Kerubananthan3761) [reply wpf/Gantt/migrate-from-sftreegrid.md]: GridSelectionMode removed.

File: wpf/Gantt/migrate-from-sftreegrid.md (hunk 3 above)
Comment (Kerubananthan3761) [reply wpf/Gantt/migrate-from-sftreegrid.md]: Renamed to GridTreeControl in all places.

File: wpf/Gantt/migrate-from-sftreegrid.md [hunk 5]
```diff
@@ -121,26 +121,26 @@ Syncfusion.Windows.Controls.Gantt.Chart</td>
 </tr>
 
 <tr>
-<td>{{'[GridTreeNodeEventHandler](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Grid.GridTreeNodeEventHandler.html)'| markdownify }}</td>
+<td>{{'[GridTreeNodeEventArgs](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Grid.GridTreeNodeEventArgs.html)'| markdownify }}</td>
```
Comment (Karthickmani97) [inline wpf/Gantt/migrate-from-sftreegrid.md:124]: Change GridTreeColumn to `SfTreeGrid columns`

Screenshot 2024-06-11 at 7 17 39 PM

**Review by Karthickmani97 — APPROVED** [review]
This is fine.

**Pull request merged**